rocAL/include/loaders/circular_buffer.h

```diff
@@ -73,8 +73,8 @@ class CircularBuffer {
     bool full();
     bool empty();
     size_t _buff_depth;
-    decoded_image_info _last_image_info;
-    std::queue<decoded_image_info> _circ_image_info;    //!< Stores the loaded images names, decoded_width and decoded_height(data is stored in the _circ_buff)
+    DecodedDataInfo _last_data_info;
+    std::queue<DecodedDataInfo> _circ_data_info;    //!< Stores the loaded data names, decoded_width and decoded_height(data is stored in the _circ_buff)
     crop_image_info _last_crop_image_info;              // for Random BBox crop coordinates
     std::queue<crop_image_info> _circ_crop_image_info;  //!< Stores the crop coordinates of the images for random bbox crop (data is stored in the _circ_buff)
     std::mutex _names_buff_lock;
```

Comment:
Commented lines 81-84, which variable is that for? It's hanging without reference

Reply:
Removed the comment